cmd/outflux/migrate_i_test.go

```diff
@@ -34,6 +34,7 @@ func TestMigrateSingleValue(t *testing.T) {
 	errs := migrate(appContext, connConf, config)
 	if errs != nil {
 		t.Error(errs[0])
+		return
```

Comment:
Seems like you just want `t.Fatal` for all of these then?

Reply:
Will change